src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from './Person';
+
+export const PeopleTable: React.FC = () => {
+  const [people, setIsPeoples] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  const getIsPeople = async () => {
```

Comment:
```suggestion
  const getPeople = async () => {
```
you aren't getting boolean value, `is` in the variable name is redundant